packages/runtime/src/widgets/ToggleButton.tsx

```diff
@@ -30,6 +30,7 @@ interface ToggleButtonPropsStyles {
   borderWidth?: number;
   selectedBorderColor?: Expression<string>;
   selectedBorderWidth?: number;
+  buttonWidth?: Expression<string>;
```

Comment:
What are you using this for? The button widths can derive their width from the overall Toggle Button width

Reply:
well it is not, in my case i wanted to apply 50 % width to two toggle buttons